### ISSUE TITLE
Requiring -f / --force as an option for purge to actually delete files

### ISSUE THREAD

**Issue by andy5995** [andy5995/rmw#33]
I saw a related topic at andreafrancia/trash-cli#32 and assumed that sooner or later someone would request that feature for rmw.

This would involve
1. the section that handles options (at the top of main()) to be updated
2. adding an extra condition for purge() to run (near the end of main())
3. Updating what's shown when --help is used (trivial_rmw.c)

@parthsuresh Would you like to attempt to add this feature?

**Comment by parthsuresh** [comment]
@andy5995 yes, I would like to work on this.

**Comment by andy5995** [comment]
When you do the PR for this, do it on the develop branch. I'll want to test it a bit before including it with the next release, and we might need to tweak how it's implemented.

**Comment by parthsuresh** [comment]
@andy5995 What should I give as the long option for '-rf' ?

**Comment by andy5995** [comment]
The same as the [GNU coreutils rm command](https://github.com/coreutils/coreutils/blob/master/src/rm.c): --recurse and --force

The short options are -r and -f. If -rf is used, it's parsed as -r and -f so be sure to keep them separated.

For rmw, I'm not sure that recurse is really necessary. For right now, if someone uses -r, just have rmw spit out a message that "-r --recurse is not implemented".

But -f option will be required to run purge()

**Comment by parthsuresh** [comment]
:+1:

**Comment by parthsuresh** [comment]
Diff for issue #33:
http://pastebin.com/6qhy2LRN

**Comment by andy5995** [comment]
@parthsuresh I'll need to make a few changes to fine tune it, but please go ahead an open a PR on the develop branch. Thank you.

**Comment by andy5995** [comment]
As it is now, purge actually runs automatically, but only once per day. If I rmw some files at 8am, it will look for files to purge. It won't do that again until after midnight the next day, unless -g is used.

-r won't be required to make this work, though originally that's what I intended when I wrote this ticket. I edited the title to reflect that.

-r is being added only for those who are used to using -rf with the GNU coreutils rm command. After your patch, instead of a user getting this message
`rmw: invalid option -- 'r'` and the help being displayed, they will get the message simply that it's not implemented.

**Comment by andy5995** [comment]
This was addressed in [this commit](https://github.com/andy5995/rmw/commit/6897b17fa0b12d740c2bc3dca50a6c894ddfeddc#diff-1677593b736bd5b60cf7f0bcb9062487) and [this one](https://github.com/andy5995/rmw/commit/8d7e58615daa7be534caec86ca02478cd60c15aa)

I used the wrong number on one of the commits, and I don't know why the other one didn't ping this ticket.
